kubernetes/tests/kubernetes/kubernetes.tests.bom

```diff
@@ -116,17 +116,7 @@ brooklyn.catalog:
           command: |
             kubectl get deployments | grep workload- | wc -l
           assertOut:
-            contains: '0'
-        - type: org.apache.brooklyn.test.framework.TestHttpCall
-          name: "14. Check Prometheus UI is Reachable"
-          url:
-            $brooklyn:formatString:
-              - "http://%s:%s"
-              - $brooklyn:entity("prometheus").attributeWhenReady("host.address")
-              - $brooklyn:entity("prometheus").attributeWhenReady("kubernetes.service.port")
-          applyAssertionTo: status
-          assert:
-          - equals: 200
+            contains: '0'        
```

Comment:
Because this is _technically_ more correct. We are testing HTTP(S) access, not SSH/shell. And also, these tests may be run against a deployed Kubernetes cluster that cannot be SSHed to, for example.